.envrc

```diff
@@ -0,0 +1,4 @@
+export VIRTUAL_ENV=."venv"
+layout python
+
+[[ -f .envrc.private ]] && source_env .envrc.private
```

Comment:
Allows custom private environment variables to be defined such as TADO username/pass for easier testing.

Reply:
we should add this to the examples and readme